manifests/rbac.yaml

```diff
@@ -0,0 +1,55 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: objectstorage-controller-role
+rules:
+- apiGroups: ["objectstorage.k8s.io"]
+  resources: ["bucketrequests", "bucketaccessrequests"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["objectstorage.k8s.io"]
+  resources: ["buckets", "bucketaccess"]
+  verbs: ["get", "list", "watch", "update", "create", "delete"]
+- apiGroups: ["objectstorage.k8s.io"]
+  resources: ["bucketclass","bucketaccessclass"]
+  verbs: ["get", "list"]
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["list", "watch", "create", "update", "patch"]
```

Comment:
A bit surprised this controller would need to `list` and `watch` `event`s.

Reply:
Added to https://github.com/kubernetes-sigs/container-object-storage-interface-controller/issues/19